destroy_cluster.sh

```diff
@@ -10,8 +10,13 @@ else
     echo "Please export CONFIG. See readme for help."
     exit 1
 fi
-CLUSTER_PREFIX=$CLUSTER_NAME
-CLUSTER_NAME=$(jq .infraID $CLUSTER_PREFIX/metadata.json | sed "s/\"//g")
+
+HASJQ="$(which jq)"
```

Comment:
You don't need to check if jq is installed, it's already a requirement per the README file.